hivemq4/base-image/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Additional build image to unpack the zip file and change the permissions without retaining large layers just for those operations
-FROM busybox:1.36.1@sha256:c3839dd800b9eb7603340509769c43e146a74c63dca3045a8e7dc8ee07e53966 AS unpack
```

Comment:
> are we changing this because of the smaller footprint?
No, footprint is not a concern here. This is only a intermediate build step.
There was just no reason to use busybox in the base-image Dockerfile and alpine in dns and k8s.

Reply:
> it also comes with unzip

It does, but this is not related to the change below.
The change below moves the installation of the unzip package from the base-image to the k8s-image, because it is only needed for the legacy operator.

Reply:
Thanks for the clarification!